custom_components/tuya_local/devices/fisher_summer_air_conditioner.yaml

```diff
@@ -0,0 +1,463 @@
+      name: current_temperature
+      type: integer
+      readonly: true
+    - id: 4
+      name: mode
+      type: string
+    - id: 5
+      name: fan_mode
+      type: string
+      mapping:
+        - dps_val: auto
+          value: Auto
+        - dps_val: mute
+          value: Quiet
+        - dps_val: low
+          value: Low
+        - dps_val: mid_low
+          value: Medium Low
+        - dps_val: mid
+          value: Medium
+        - dps_val: mid_high
+          value: Medium High
+        - dps_val: high
+          value: High
+        - dps_val: strong
+          value: Strong
+secondary_entities:
+  - entity: switch
+    name: Power
+    category: config
+    icon: "mdi:power"
+    dps:
+      - id: 1
+        name: switch
+        type: boolean
+  - entity: number
+    name: Target Temperature
+    category: config
+    class: temperature
+    dps:
+      - id: 2
+        name: value
+        type: integer
+        unit: C
+        range:
+          min: 160
+          max: 310
+        mapping:
+          - scale: 10
+            step: 5
+  - entity: sensor
+    name: Current Temperature
+    class: temperature
+    dps:
+      - id: 3
+        name: sensor
+        type: integer
+        unit: C
+        readonly: true
+  - entity: select
+    name: Mode
+    category: config
+    dps:
+      - id: 4
+        name: option
+        type: string
+        mapping:
+          - dps_val: auto
+            icon: mdi:thermostat-auto
+            value: heat_cool
+          - dps_val: cold
+            icon: "mdi:snowflake"
+            value: cool
+          - dps_val: wind
+            icon: "mdi:fan"
+            value: fan_only
+          - dps_val: wet
+            icon: "mdi:water"
+            value: dry
+          - dps_val: hot
+            value: heat
+            icon: "mdi:heat-wave"
+  - entity: select
+    name: Fan Mode
+    category: config
+    dps:
+      - id: 5
+        name: option
+        type: string
+        mapping:
+        - dps_val: auto
+          value: Auto
+        - dps_val: mute
+          value: Quiet
+        - dps_val: low
+          value: Low
+        - dps_val: mid_low
+          value: Medium Low
+        - dps_val: mid
+          value: Medium
+        - dps_val: mid_high
+          value: Medium High
+        - dps_val: high
+          value: High
+        - dps_val: strong
+          value: Strong
+  - entity: sensor  # no sensor in Fisher Summer AC, returns 0
+    name: Current Humidity
+    class: humidity
+    dps:
+      - id: 18
+        name: sensor
+        type: integer
+        unit: "%"
+        readonly: true
+        optional: true
+  - entity: binary_sensor
+    name: Fault
+    class: problem
+    category: diagnostic
+    dps:
+      - id: 20
+        type: bitfield
+        name: sensor
+        mapping:
+          - dps_val: 0
+            value: false
+          - value: true
+  - entity: sensor  # no sensor in Fisher Summer AC, returns 0
+    name: PM 2.5
```

Comment:
sensors with class do not require a name, unless there are multiple of them or the class default name does not fit the sensor.